requires.py

```diff
@@ -242,6 +242,13 @@ def enable_network_management(self):
         """
         self._request({'enable-network-management': True})
 
+    def enable_loadbalancer_management(self):
+        """
+        Request the ability to manage networking.
+        """
+        self._request({'enable-loadbalancer-management': True})
```

Comment:
I see this new request to enable loadbalancer management on the requires side, but I don't see any corresponding code  being added to the provides side. Shouldn't there be something like `IntegrationRequest.requested_loadbalancer_management` to go with this?